lesson36/src/js/article.js

```diff
@@ -0,0 +1,114 @@
+const articleList = document.getElementById("js-article-List");
+
+const createElementWithClassName = (type, className) => {
+  const element = document.createElement(type);
+  element.className = className;
+  return element;
+};
```

Comment:
This function is very useful!
I learned a lot.

Reply:
Very useful!
I had forgotten about it, as I knew when I saw Yuka-san's code ...!
Thank you!